### ISSUE TITLE
TypeError: Body is unusable

### ISSUE THREAD

**Issue by mirko314** [blomqma/next-rest-framework#66]
Hello, 
while integrating the framework into an app-router based project, I encountered an bug:

```
Next REST Framework encountered an error:
TypeError: Body is unusable
```
Reproduction:
- Create a POST handler that tries to read the body like `const { foo, bar } = await req.json();`

Sample call (`apps/example/src/app/api/routes/todos/route.ts`):
```
POST: {
    input: {
      contentType: 'application/json',
      body: z.object({
        foo: z.string(),
        bar: z.number()
      })
    },
    output: [
      {
        status: 201,
        contentType: 'application/json',
        schema: z.object({
          foo: z.string(),
          bar: z.number(),
          query: z.object({
            test: z.string()
          })
        })
      }
    ],
    handler: async (req) => {
      const { foo, bar } = await req.json();
      console.log('foo', foo);
      return NextResponse.json(
        { foo, bar, query: { test } },
        {
          status: 201
        }
      );
    }
  },
  ```
It seems to be an issue, that occurs when the json body is read twice: https://stackoverflow.com/questions/75071352/reading-the-body-of-a-request-in-hooks-server-in-sveltekit

**Comment by blomqma** [comment]
Thanks for reporting, I have fixed this now in the latest release [v1.2.1](https://github.com/blomqma/next-rest-framework/releases/tag/v1.2.1).

**Comment by mirko314** [comment]
Wow, that was super fast, thanks a bunch!
